server/controllers/routes/post/getDraftPosts.js

```diff
@@ -0,0 +1,31 @@
+const { getDraftEvents, getDraftPublicServices } = require('../../../database/queries/getDraftPosts');
+
+draftPosts = async (req, res) => {
+  try {
+    const { id: publisher_id } = req.user;
+    const resEvent = await getDraftEvents(publisher_id)
+    const resPublic = await getDraftPublicServices(publisher_id)
+    let allPosts = [];
+    await resEvent.rows.forEach( async (element) => {
```

Comment:
You don't need lines 9-14
You could concat the rows immediately.
const allPosts = [...resEvent.rows, ...res.Public.rows]